### PR TITLE
Weather-App-Review

### PR DESCRIPTION
**Pull request by AysamDev**
can't wait to hear your review

### CODE REVIEW

File: dist/APIManager.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+
+    constructor(renderer)
+     {
+         this.cityData = []
+         this.renderer = renderer
```
Comment (ameliawalcek) [inline dist/APIManager.js:8]: You should not use your renderer in here. Try to make your code MVC. The main.js talks to the manager, then the manager sends back the data, and then the main.js sends that data to the renderer. The Manager should not have anything to do with the renderer.

File: dist/main.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const renderer = new Renderer()
+const api = new APIManager(renderer)
+
+const loadPage = function()
+{
+    api.getDataFromDB()
```
Comment (ameliawalcek) [inline dist/main.js:6]: Call it instead `getCities` it describes the method better and makes reading your code easier

File: dist/main.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+const handleSearch = async function()
+{
+    let cityName = $('#city-input').val()
+    if(cityName != "")
```
Comment (ameliawalcek) [inline dist/main.js:12]: `!==`

File: dist/main.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        $('#city-input').val('')
+        api.getCityData(cityName)
```
Comment (ameliawalcek) [inline dist/main.js:15]: you would async await the data and then send it to the renderer from this file.

File: server/models/City.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose')
+mongoose.connect(process.env.URL ||'mongodb://localhost/Weather', { useNewUrlParser: true })
+
+const Schema = mongoose.Schema
+
+const citySchema = new Schema({
```
Comment (ameliawalcek) [inline server/models/City.js:6]: make a separate file for your schema. it should not be in the server

File: server/routes/api.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+router.post("/city",async(req,res)=>
+{
+   let city =  new City(req.body)
+   const save = await city.save()   
+   res.send("new city was added successfully") 
+})
+
+router.delete("/city/:cityName",async(req,res)=>
+{
+   let city =  req.params.cityName
```
Comment (ameliawalcek) [inline server/routes/api.js:39]: deconstruct your code instead. It looks better `const { cityName } = req.params`

**Review by ameliawalcek — COMMENTED** [review]
Really nice job. I left you some comments. 

Make sure next project you do, do not send your data from the manager to the renderer. I left a comment about what you should do instead

File: dist/APIManager.js (hunk 1 above)
Comment (AysamDev) [reply dist/APIManager.js:8]: okay

**Pull request merged**